### PR TITLE
Add file-based swifttool tasks, make overrideable

### PR DESCRIPTION
**Pull request by timrchavez**

### CODE REVIEW

File: roles/swift-common/defaults/main.yml [hunk 1]
```diff
@@ -78,3 +78,7 @@ swift:
       net.ipv4.tcp_syncookies: 0
       net.ipv4.tcp_tw_recycle: 1
       net.ipv4.tcp_tw_reuse: 1
+  # NOTE: Make sure you update the swifttool version in both places
+  swifttool_url: https://github.com/blueboxgroup/swifttool/archive/v0.0.1.tar.gz
+  swifttool_version: 'v0.0.1'
+  swifttool_method: 'git'
```
Comment (masteinhauser) [inline roles/swift-common/defaults/main.yml]: why switch to using the underscore var style instead of the hash?

Comment (timrchavez) [inline roles/swift-common/defaults/main.yml]: Just following "convention" (used novnc as an example)...

**Comment by timrchavez** [comment]
retest

**Comment by j2sol** [comment]
retest

**Comment by j2sol** [comment]
Good, probably need to backport this to the RELEASE-2.0.x branch.

**Pull request merged**